src/pages/fallback/CommonFeedbackMessage.tsx

```diff
@@ -3,17 +3,34 @@ import { Trans } from 'react-i18next'
 
 import { GeyserFeedbackFromUrl } from '../../constants'
 
-export const CommonFeedbackMessage = () => {
+export const CommonFeedbackMessage = ({prefix}: {prefix: string | ''}) => {
```

Comment:
it might be better to use `prefix?: string` incase it might be empty, having empty '' feels off. That way, you wouldn't have to send in prefix={''}, you can simply skip having it as a prop.